src/ai/backend/common/web/session/__init__.py

```diff
@@ -35,7 +36,7 @@ class _CookieParams(TypedDict, total=False):
     secure: Optional[bool]
     httponly: bool
     samesite: Optional[str]
-    expires: str
+    expires: Optional[str]
```

Comment:
This name is also confusing.
It is hard to understand the usage of `expires` and `login_session_extend_time`.

Reply:
expires is as mentioned in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires), the date/time of response is expired. Therefore I don't think it need to be changed.